docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1109 @@
+
+    Example authority_hints addition:
+
+    .. code-block:: json
+
+        {
+          "iat": 1718207217,
+          "exp": 1749743216,
+          "iss": "https://credentials.example.gov",
+          "sub": "https://credentials.example.gov",
+          "authority_hints": ["https://trust-anchor.example.gov"],
+          //...
+        }
+
+    Example Federation JWK with certificate chain:
+
+    .. code-block:: json
+
+        {
+          "kid": "NsXymfIILEPR5Y0t",
+          "kty": "EC",
+          "x": "gXY4FApFJCj91Gpb1K9GEIouTq2X3L0K64Iq0ob4l_g",
+          "y": "l-6dcrIrFVdrzoY9cRJv9zNuFOR3MsDz6TSDhB0xEo4",
+          "crv": "P-256",
+          "x5c": [
+            "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",
+            "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."
+          ]
+        }
+
+    Example Protocol JWK with extended certificate chain:
+
+    .. code-block:: json
+
+        {
+          "kid": "ProtocolKeyId123",
+          "kty": "EC",
+          "x": "protocol_key_x_coordinate...",
+          "y": "protocol_key_y_coordinate...",
+          "crv": "P-256",
+          "x5c": [
+            "MIIDprotocolCert...",  // Protocol cert (signed by Federation key)
+            "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",  // Federation cert
+            "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."   // Root cert
+          ]
+        }
+
+  2. **Publish Updated Entity Configuration**: Publish the updated EC at the ``/.well-known/openid-federation`` endpoint as specified in :ref:`trust:The Infrastructure of Trust`.
+
+  3. **Submit Resolve Request**: Call the **Trust Anchor**'s ``/resolve`` endpoint (as defined in :ref:`trust:Trust Infrastructure Requirements`) with URL-encoded parameters:
+
+    - ``sub``: Federation Entity Identifier.
+    - ``trust_anchor``: **Trust Anchor** Federation Entity Identifier (always the root Trust Anchor, even for Intermediate-mediated onboarding).
+
+    Example resolve request:
+
+    .. code-block:: http
+
+        GET /resolve?sub=https%3A%2F%2Fcredentials.example.gov&trust_anchor=https%3A%2F%2Ftrust-anchor.example.gov HTTP/1.1
+        Host: trust-anchor.example.gov
+
+**Step 4: Resolve Response and Onboarding Completion**
+
+Following the resolve request, the **Federation Authority** performs:
+
+  - **Trust Chain Reconstruction**: Reconstruction of a valid Trust Chain for the entity as defined in :ref:`trust:The Infrastructure of Trust`.
+  - **Federation Trust Mark Generation**: Generation of an IT-Wallet Federation Trust Mark as a signed JWT attestation of the entity's federation membership and compliance with IT-Wallet technical requirements.
+  - **Trust Mark Integration in Subordinate Statement**: The generated Trust Mark is included in the entity's Subordinate Statement as defined in :ref:`trust:Subordinate Statements`.
```

Comment:
this is not encouraged in openid federation draft 43, since trust marks SHOULD only be provided within the ECs.